README.md

```diff
@@ -101,21 +101,13 @@ key is in this repo under [deb.asc](/deb.asc).
 To install eza from this repo use:
 
 ```bash
-wget -qO- https://raw.githubusercontent.com/eza-community/eza/main/deb.asc | sudo tee /etc/apt/trusted.gpg.d/gierens.asc
-echo "deb http://deb.gierens.de stable main" | sudo tee /etc/apt/sources.list.d/gierens.list
+sudo mkdir /etc/apt/keyrings
+wget -qO- https://raw.githubusercontent.com/eza-community/eza/main/deb.asc | gpg --dearmor | sudo dd of=/etc/apt/keyrings/gierens.gpg
```

Comment:
Can you change this to `| sudo gpg --dearmor -o /etc/apt/keyrings/gierens.gpg` ... This avoids using `dd` here.

Reply:
done! also added a `chmod o+r`. on a weird system , I had a umask of 007, and even `sudo apt update` didn't like it when the file wasn't readable. guess apt runs as the `_apt` user or something

Reply:
It seems you forgot the 644, I've added it

Reply:
whoops. thanks!